MyWeightTests/UITableViewExtensionsTests.swift

```diff
@@ -0,0 +1,162 @@
+//
+//  UITableViewExtensionsTests.swift
+//  MyWeight
+//
+//  Created by Bruno Koga on 22/06/16.
+//  Copyright © 2016 Diogo Tridapalli. All rights reserved.
+//
+
+import XCTest
+@testable import MyWeight
+
+class InvarianteTableViewCell: UITableViewCell { }
+class EmojiTableViewCell😎: UITableViewCell { }
+class a0_$WÅ´Ø: UITableViewCell { }
+
+class UITableViewExtensionsTests: XCTestCase {
+    private var tableView: UITableView!
+    
+    override func setUp() {
+        super.setUp()
+        tableView = UITableView()
+    }
+    
+    override func tearDown() {
+        tableView = nil
+        super.tearDown()
+    }
+    
+    func testDefaultReuseIdentifier() {
+        var cellClass = UITableViewCell.self
+        var cellDefaultReuseIdentifier = cellClass.defaultReuseIdentifier
+        var expectedDefaultReuseIdentifier = "UITableViewCell"
+        XCTAssertEqual(cellDefaultReuseIdentifier, expectedDefaultReuseIdentifier)
```

Comment:
I'm using [Expecta](https://github.com/specta/expecta) in Objective-C and my tests are much friendlier, what do you think about trying [Nibmle](https://github.com/quick/nimble)?
It will be something like: `expect(cellDefaultReuseIdentifier).to(equal(expectedDefaultReuseIdentifier))`